cluster/addons/podsecuritypolicies/kube-system-binding.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: kube-system:default-binding
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kube-system:default:podsecuritypolicy
+subjects:
+- apiVersion: v1
+  kind: ServiceAccount
+  name: default
+  namespace: kube-system
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: kube-system:default-binding
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: kube-system:default:podsecuritypolicy
+subjects:
+- apiVersion: rbac.authorization.k8s.io
+  kind: Group
+  name: system:serviceaccounts:kube-system
```

Comment:
this gives the privileged PSP to all service accounts in the kube-system namespace, which includes the ones used to run controller loops, which effectively gives it to all users